src/components/views/dialogs/InteractiveAuthDialog.js

```diff
@@ -72,19 +72,20 @@ export default React.createClass({
         let content;
         if (this.state.authError) {
             content = (
-                <div>
-                    <div>{ this.state.authError.message || this.state.authError.toString() }</div>
+                <div id='mx_Dialog_content'>
+                    <div role={(this.state.authError.message || this.state.authError.toString()) ? "alert" : ""}>{ this.state.authError.message || this.state.authError.toString() }</div>
```

Comment:
I think the existence of this `div` should be conditional on whether there is a message to display. That way, we only have a single conditional and the `role` can always be "alert".

Reply:
In the ErrorDialog the condition checking the error message turned out to be redundant as the same check is performed a few lines above. But thanks for the idea, I have checked the other occurences of this too.